community/common/src/main/java/org/neo4j/kernel/api/security/AccessMode.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.api.security;
+
+/** Controls the capabilities of a KernelTransaction. */
+public interface AccessMode
+{
+    enum Static implements AccessMode
+    {
+        /** No reading or writing allowed. */
+        NONE
```

Comment:
Strictly speaking, is this `NONE` or `PASSWORD_CHANGE_ONLY`? If the latter, we should name it as such. If the former, what is the value of being allowed in with absolutely no access, as opposed to not being allowed in?